### ISSUE TITLE
Multiblock Controller

### ISSUE THREAD

**Issue by raoulvdberge** [refinedmods/refinedstorage2#82]

**Comment by BooBooFastCat1** [comment]
At the moment in milestone 2.14, the controllers can be multiple per network and pass the signal through but not power is this intentional or just a prelude to a proper implementation in milestone 3?

**Comment by raoulvdberge** [comment]
Correct! Later, multiple controllers will form a multiblock, whereas in milestone 2.14 they are separate blocks.

**Comment by raoulvdberge** [comment]
Not for milestone 3 though (notice the removal from the milestone last year). This feature is for after RS 2 is released properly.
